pkg/minikube/download/image.go

```diff
@@ -224,6 +226,35 @@ func ImageToCache(img string) error {
 	}
 }
 
+func ImageFromGitHubToCache(img, imgVersion string) (string, error) {
```

Comment:
this needs to have a comment that says what it does... and why is it exported? is it used somwhere else